src/stores/GlobalStore.ts

```diff
@@ -67,7 +66,8 @@ export const useGlobalStore = defineStore('globalStore', {
     },
     actions: {
         init() {
-            this.connection = new Connection(this.rpc.url)
+            console.log(this.rpc.url.toString())
```

Comment:
is this console log left by purpose? if it's not needed, should be removed

Reply:
done